src/util/eventfilter_parser/eventfilter_parser_remove_compiler_warnings.txt

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
What is the intended purpose of this file?

Reply:
The compiler throws some errors when using the autogenerated eventfilter_parser_grammar.c file from the eventfilter_parser_grammar.leg. This file should give some hints on how to correct the compiler errors.